README.md

```diff
@@ -57,6 +57,8 @@ Your team also needs you to query some information from the database that you ha
 
 2) I want to find out the top 3 car manufacturers that customers bought by sales (quantity) and the sales number for it in the current month.
 
+*Note: please submit screenshots of the sql query results.*
```

Comment:
Since there is no sample data, how would they return the query results?

Reply:
noted. Removed.